chunk_downloader.go

```diff
@@ -458,6 +458,13 @@ func decodeChunk(ctx context.Context, scd *snowflakeChunkDownloader, idx int, bu
 			return nil
 		}
 		highPrec := higherPrecisionEnabled(scd.ctx)
+		if scd.sc == nil {
```

Comment:
What do you think about:
```
logger.WithContext(ctx).Debugf("scd.sc is nil? %v, scd.sc.cfg is nil? %v", scd.sc == nil, scd.sc != nil && scd.sc.cfg == nil)
```

If you want to stay with your version, please add TODO to remove it later :)